xLights/models/ArchesModel.cpp

```diff
@@ -629,8 +644,15 @@ void ArchesModel::ImportXlightsModel(wxXmlNode* root, xLightsFrame* xlights, flo
         wxString psp = root->GetAttribute("PixelSpacing");
         wxString ls = root->GetAttribute("LayerSizes");
         wxString h = root->GetAttribute("Hollow");
+        wxString zz = root->GetAttribute("ZigZag");
 
         // Add any model version conversion logic here
+
+        if (zz == "")
```

Comment:
Should be able to just add "true" as a second parameter to the GetAttribute call and delete this whole if.

Reply:
Sounds like what happens to me at work.  I tell them "hey I don't create anything new I copy from what you guys already have in the code base and then you tell me its wrong"....lol.

For StrandsZigZagOnString yeah I'd delete it if you don't see it being used.  I did a quick search on GitHub and don't see the function actually used.  Doesn't seem to make sense for an entire model class to have it defined cause users are free to wire things however they like.

I was going to complain about the zigzag being stored as "true" instead of "1" because it takes more bytes but I searched and it seems like its been done that way elsewhere.

Reply:
Some developers like to store all the settings as strings to make it easier to view in the XML files. I'm not a fan of using strings as from a performance standpoint they are bad, they also are bad from a maintenance/"easy to create bug" standpoint. I prefer strongly typed variables like bool or int/enum for settings.

Reply:
XML files don't store anything else?  Maybe there are reasons why JSON is so popular for new projects.

Reply:
Yeah I thought everything gets turned into a string with the XML stuff then we convert it back when reading it in.  I mainly followed what was there when I started.  I think all the developers would like to get away from the way xLights used the "ModelXML" object carrying around XML and using it to read/write object status throughout the program.  I think we could gain a bit of speed if we read the XML or switch to JSON (which I'm not that familiar with) and then the object status is held in class object variables until we need to write back out to Save things.

Reply:
Is there a suggestion you want implemented for the PR?  Or a suggestion you'd like to see in a follow-up, broader PR?

FWIW, on my setup, the thing that is the slowest is building a couple of trees... I am sure I can fix those without a huge overhaul of the XML stuff.

Reply:
No that conversation was for the future.  This PR is good to go I was just at work yesterday and didn't look at it till now.  Thanks for reminding me.